bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Flight/Maneuver.cpp

```diff
@@ -0,0 +1,268 @@
+            Stores information and metadata about a finite maneuver that will be or has been performed.
+        )doc"
+    )
+
+        .def(
+            init<const Array<Instant>&, const Array<Vector3d>&, const Shared<const Frame>&, const Array<Real>&>(),
+            arg("instants"),
+            arg("acceleration_profile"),
+            arg("frame"),
+            arg("mass_flow_rate_profile"),
+            R"doc(
+                Constructor.
+
+                Args:
+                    instants (list[Instant]): An array of instants, must be sorted.
+                    acceleration_profile (list[numpy.ndarray]): An acceleration profile of the maneuver, one numpy.ndarray per instant.
+                    frame (Frame): A frame in which the acceleration profile is defined.
+                    mass_flow_rate_profile (list[float]):  A mass flow rate profile of the maneuver, one float per instant.
+            )doc"
+        )
+
+        .def("__str__", &(shiftToString<Maneuver>))
+        .def("__repr__", &(shiftToString<Maneuver>))
+
+        .def(
+            "is_defined",
+            &Maneuver::isDefined,
+            R"doc(
+                Check if the maneuver is defined.
+
+                Returns:
+                    bool: True if the maneuver is defined, False otherwise. (Always returns true).
+            )doc"
+        )
+        .def(
+            "get_instants",
+            &Maneuver::getInstants,
+            R"doc(
+                Get the instants.
+
+                Returns:
+                    list[Instant]: The instants.
+            )doc"
+        )
+        .def(
+            "get_acceleration_profile",
+            &Maneuver::getAccelerationProfile,
+            arg("frame") = Maneuver::DefaultAccelFrameSPtr,
+            R"doc(
+                Get the acceleration profile.
+
+                Args:
+                    frame (Frame, optional): The frame in which the acceleration profile is defined. Defaults to the default acceleration frame.
+
+                Returns:
+                    list[numpy.ndarray]: The acceleration profile.
+            )doc"
+        )
+        .def(
+            "get_mass_flow_rate_profile",
+            &Maneuver::getMassFlowRateProfile,
+            R"doc(
+                Get the mass flow rate profile.
+
+                Returns:
+                    list[float]: The mass flow rate profile.
+            )doc"
+        )
+        .def(
+            "get_start_instant",
+            &Maneuver::getStartInstant,
+            R"doc(
+                Get the start instant of the maneuver.
+
+                Returns:
+                    Instant: The start instant.
+            )doc"
+        )
+        .def(
+            "get_end_instant",
+            &Maneuver::getEndInstant,
+            R"doc(
+                Get the end instant of the maneuver.
+
+                Returns:
+                    Instant: The end instant.
+            )doc"
+        )
```

Comment:
Maybe store these an interval instead of start and end instant?

Reply:
They're not stored as interval, they're stored as an array of instants, but yes I can return an interval